### ISSUE TITLE
Calling launch in kotlinjs causes a TypeError to be logged

### ISSUE THREAD

**Issue by ansman** [Kotlin/kotlinx.coroutines#340]
If you compile the following code and execute it:
```kotlin
fun main(args: Array<String>) {
  launch {}
}
```
You get the following exception printed to the terminal:
```
TypeError: throwCCE is not a function
```

**Comment by qwwdfsad** [comment]
I can't reproduce this problem on my machine.
Could you please provide more details (node.js version, Kotlin version, build script etc.) or self-contained project example?

**Comment by ansman** [comment]
Sure, no problem. I've attached a same project.

Just cd in to the dir and run `npm run bug`

[coroutines-bug.zip](https://github.com/Kotlin/kotlinx.coroutines/files/1935819/coroutines-bug.zip)

**Comment by ansman** [comment]
Seems like it could be solved by fixing #339 then

**Comment by SUPERCILEX** [comment]
@elizarov any idea when this will be released? Or are there snapshot builds we could use? Coroutines are basically unusable in JS, seems like a big deal. 😕
